readthedocs/templates/projects/project_import.html

```diff
@@ -209,6 +209,26 @@ <h1>{% trans "Import a Repository" %}</h1>
         </div>
       {% endblock %}
 
+      {# accounts is a dictionary with the provider as key and a list of connected accounts of that provider #}
+      {% get_social_accounts user as accounts %}
+      {% if accounts %}
+        <h3>{% trans "Filter by Own Repositories" %}</h3>
```

Comment:
If you choose to combine, make sure to specify that the organization comes from a specific social account. For example, "Read the Docs (github)" or the like.

Reply:
I'm not sure about the idea of combining them from a UX perspective. I think it's too much clear for the user if we have two sections. Maybe "My repositories" and "My organizations"?

Also I started trying to make just one list and I think it complicates the code a little more.

Besdies, by adding the ` (social account)` to each item in the list will be too verbose I think. Don't you think? I'd say that it will be rare to have a two organizations with the same name under different services. Or maybe, we can add the name of the social account only if this happens? (that will be more complicated to code).

Just thinking aloud. I don't have an strong opinion yet and I will probably change my mind while I write the code since knockout is new to me :)